pkg/apis/keycloak/v1alpha1/keycloak_types.go

```diff
@@ -87,6 +87,10 @@ type KeycloakSpec struct {
 	// objects and users will have to create them manually, if needed.
 	// +optional
 	DisableMonitoringServices bool `json:"DisableDefaultServiceMonitor,omitempty"`
+	// Specify whether disabling the syncing of instances from the Keycloak CR to the statefulset replicas
+	// should be enabled or disabled. Defaults to false.
```

Comment:
Maybe should we mention it's particularly useful for HPAs?

Reply:
Thanks @vmuzikar! good idea